src/main/java/uk/gov/ons/fwmt/fwmtrmadapter/message/impl/JobServiceProducerImpl.java

```diff
@@ -26,21 +28,23 @@
   @Autowired
   private Exchange exchange;
 
-  public void sendCreateJobRequest(FWMTCreateJobRequest createJobRequest) {
+  @Autowired
+  private ObjectMapper objectMapper;
+
+
+  public <T> void sendMessage(Object dto) {
     try {
-      String JSONJobRequest = convertToJSON(createJobRequest);
+      String JSONJobRequest = convertToJSON(dto);
       rabbitTemplate.convertAndSend(exchange.getName(), "job.svc.job.request.create", JSONJobRequest);
     } catch(JsonProcessingException e) {
       e.printStackTrace();
     }
-    log.info("Done!");
-
+    log.info("Message send to queue", dto);
   }
 
-  private String convertToJSON(FWMTCreateJobRequest createJobRequest) throws JsonProcessingException {
+  private <T> String convertToJSON(Object dto) throws JsonProcessingException {
 
-    ObjectMapper mapper = new ObjectMapper();
-    String JSONJobRequest = mapper.writeValueAsString(createJobRequest);
+    String JSONJobRequest = objectMapper.writeValueAsString(dto);
     log.info(JSONJobRequest);
```

Comment:
Log info with String at front to differentiate.
for e.g.: log.info("CreateJobRequest :==>> {}", JSONJobRequest);